internal/connector/pingone/sso/pingone_sso_connector.go

```diff
@@ -57,6 +57,7 @@ func (c *PingOneSSOConnector) Export(format, outputDir string, overwriteExport b
 		resources.PasswordPolicy(&c.clientInfo),
 		resources.Population(&c.clientInfo),
 		resources.PopulationDefault(&c.clientInfo),
+		resources.PopulationDefaultIdp(&c.clientInfo),
```

Comment:
The resource also needs to be added to the `pingone_sso_connector_test.go` file for terraform plan integration testing.